File: cart-service/src/main/java/com/sudo248/cartservice/controller/dto/ProductDto.java
package com.sudo248.cartservice.controller.dto;

import com.sudo248.cartservice.controller.dto.SupplierProductDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductDto implements Serializable {
    private String productId;
    private String name,description,sku;
    private List<ImageDto> images;
}

File: client/SocStaff/app/src/main/java/com/sudo248/soc_staff/data/dto/discovery/ProductDto.kt
package com.sudo248.soc_staff.data.dto.discovery

data class ProductDto(
    val productId: String = "",
    val name: String,
    val description: String,
    val sku: String = "",
    val images: List<ImageDto>? = null,
    val categoryIds: List<String> = listOf(),
    val supplierProducts: List<SupplierProductDto> = listOf(),
)